products/osconfig/api.yaml

```diff
@@ -838,3 +841,704 @@ objects:
                    One day of the month. 1-31 indicates the 1st to the 31st day. -1 indicates the last day of the month.
                    Months without the target day will be skipped. For example, a schedule to run "every month on the 31st"
                    will not run in February, April, June, etc.
+  - !ruby/object:Api::Resource
+    name: 'GuestPolicies'
+    base_url: "projects/{{project}}/guestPolicies"
+    create_url: "projects/{{project}}/guestPolicies?guestPolicyId={{guest_policy_id}}"
+    update_verb: :PATCH
+    self_link: "{{name}}"
```

Comment:
Could we use the `projects/{project_number}/guestPolicies/{guestPolicyId}` form here instead? I think the import format is easier to understand that way. Users are sometimes confused when the format of the import is the `name` field.